LibertyDSNP/example-client
# Feature/create wallet api 177602979

Purpose
---------------
As a developer I want a singular wallet API to integrate into the example client as opposed to having to worry about separate implementations for multiple wallets.


Solution
---------------
Create a singular wallet API that implements common methods we need from a wallet and handles the specific communication with those wallets and their APIs.


Change summary
---------------
* Created Wallet api interface that all wallet's implements
* Added current wallet to userSlice of redux
* Implemented Metamask wallet to demonstrate wallet api
* Added second login option for Metamask
* Fixed tests to account for new interface


Steps to Verify
----------------
1. Login with Torus
2. logout
3. login with Metamask
4. logout
5. All steps should work. And verify with wallet address
